### DISCUSSION TITLE
I'll get this error

### DISCUSSION THREAD

**Discussion by AiDarkEzio** [fent/node-ytdl-core#1206] [Q&A] [answered]
evalmachine.<anonymous>:25
[])||(0,c[2])((0,c[45])((0,c[25])(c[21],c[55],(0,c[11])()),c[2],(0,c[86])(c[55],c[53]),c[13],c[77],c[27]),c[52],c[62],c[74]),1==c[43]&&((0,c[45])((0,c[2])((0,c[13])(c[65+Math.pow(3,2)%391],c[85]),c[78],c[35],c[68]),c[2],((0,c[76])(c[3],c[77]),c[40])(c[77]),c[40],c[87]),1)||(((((0,c[13])(c[87],c[65]),c[52])(c[46],c[Math.pow(5,3)%149+-51]),c[47])(c[63],c[1]),c[62])(c[50],c[18]),c[28])(c[56],c[50])};lma(ncode);

                                      ^

SyntaxError: Missing catch or finally after try
    at new Script (node:vm:100:7)
    at exports.decipherFormats (E:\x\node_modules\ytdl-core\lib\sig.js:116:51)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Promise.all (index 0)
    at async exports.getInfo (E:\x\node_modules\ytdl-core\lib\info.js:401:17)

**Comment by ArmoMan** [comment, marked as answer]
I got the same error. In node_modules/ytdl-core/lib/sig.js file, you can comment out: `const nTransformScript = functions.length > 1 ? new vm.Script(functions[1]) : null;` and you can delete nTransformSctipt from: `exports.setDownloadURL(format, decipherScript, nTransformScript);`

It might affect some features, but you will not get an error.

Heres is the full edit of what I did:
`exports.decipherFormats = async(formats, html5player, options) => {`
`  let decipheredFormats = {};`
`  let functions = await exports.getFunctions(html5player, options);`
 ` const decipherScript = functions.length ? new vm.Script(functions[0]) : null;`
`  // const nTransformScript = functions.length > 1 ? new vm.Script(functions[1]) : null;`

`  formats.forEach(format => {`
      `// exports.setDownloadURL(format, decipherScript, nTransformScript);`
      `exports.setDownloadURL(format, decipherScript);`
      `decipheredFormats[format.url] = format;`
`  });`
  `return decipheredFormats;`
`};`

**Comment by TimeForANinja** [comment]
closed and fixed with the latest release
